### ISSUE TITLE
Swap across/down plugin corrupts puzzle grid

### ISSUE THREAD

**Issue by jpd236** [mrichards42/xword#33]
The swap across/down plugin seems to be broken, at least on Mac.

One issue is that it calls a "SwapDirection" function which appears to have been renamed to "InvertDirection" in https://github.com/mrichards42/xword/commit/cb8e5c8780f4e931a6073dcb139d085faf4f559f.

However, even with that resolved, the grid ends up in a broken state after the swap. Grid numbers shift around as you click through the grid; the selected square doesn't always get highlighted; filled in answers can disappear.

(Given that this plugin folder begins with a "_", it seems like it might have been shelved and thus bitrotted... but I don't see anything at first glance that actually strips out such plugins from release builds. Will file a separate bug for that).

**Comment by mrichards42** [comment]
Yeah, it's definitely old and broken.  I think this was written when the puz library was a lot simpler.  I'm sure it would be possible to make it work again, but the novelty wore off pretty quickly, and I didn't bother.
